common/djangoapps/student/views.py

```diff
@@ -1427,8 +1426,15 @@ def _do_create_account(form):
 
     Note: this function is also used for creating test users.
     """
-    if not form.is_valid():
-        raise ValidationError(form.errors)
+    errors = {}
+    form.is_valid()
```

Comment:
Using a method called `is_something` to achieve side effects just looks wrong to me. The Django 1.8 docs [say](https://docs.djangoproject.com/en/1.8/ref/forms/api/#django.forms.Form.errors) "You can access errors without having to call is_valid() first. The form’s data will be validated the first time either you call is_valid() or access errors." So can we remove these `is_valid()` calls?